extensions/smallrye-metrics/deployment/src/main/java/io/quarkus/smallrye/metrics/deployment/SmallRyeMetricsProcessor.java

```diff
@@ -154,7 +153,7 @@ MetricsCapabilityBuildItem metricsCapabilityBuildItem() {
     @Record(STATIC_INIT)
     void createRoute(BuildProducer<RouteBuildItem> routes,
             SmallRyeMetricsRecorder recorder,
-            HttpRootPathBuildItem httpRoot,
+            FrameworkRootPathBuildItem frameworkRoot,
```

Comment:
So, does that mean that Framework is equivalent to NonApplication? 
Can't we have a single word?

Reply:
Yeah, I kind of flip flopped between them.

Leaning towards "NonApplication". If that's preferred I can align them all

Reply:
Yes, I actually prefer avoiding the word "framework"

Reply:
How about `Core`?

Reply:
Um, actually I think something like `management` would make more sense

Reply:
`management` implies you can actually manage something, which isn't the case with these endpoints.

I've changed it to "NonApplication"